XiEditor/StatusBar.swift

```diff
@@ -82,11 +82,12 @@ class StatusBar: NSView {
     // Difference (in points) to compensate for when status bar is resized
     let minWidthDifference: CGFloat = 3
 
-    // Returns the minimum width required to display all items.
+    // Returns the minimum width required to display items without
+    // clipping in the status bar.
     var minWidth: CGFloat {
         return currentItems.values.filter { $0.isHidden == false }
             .map({$0.bounds.width})
-            .reduce(CGFloat(currentItems.count - 1) * statusBarPadding, +)
+            .reduce(CGFloat(currentItems.count - hiddenItems.count - 1) * statusBarPadding, +)
```

Comment:
I think this should have braces? Subtraction isn't associative, so `(currentItems.count - hiddenItems.count) - 1 != (currentItems.count - (hiddenItems.count - 1)`

(I'm sure the expected behaviour is defined in the swift spec, but braces make it easier for the code reviewer)